### DISCUSSION TITLE
Add "All Authors" and "Posts by Author" to ease list posts by author in blog

### DISCUSSION THREAD

**Discussion by Ouch1978** [facebook/docusaurus#5041] [Ideas]
I'm building an internal knowledge base with Docusaurus V2, which contains both docs and blog.

Formerly, our blog was built with jekyll, which supports multiple authors with the following functions:
1. An "All Authors" page to list all authors and his/her posts.
2. An "Posts by Author" page to list all posts that published by a certain author.

With these two function, we could find blog posts by a certain author easier.

**Comment by slorber** [comment]
Agree. I suggest posting this to [our Canny board](https://docusaurus.io/feedback/) so that we figure out how much interest there is for this feature.
